### PR TITLE
Add args on phpunit in makefile

### PR DESCRIPTION
**Pull request by JeremieSamson**
In order to pass phpunit args like --filter

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -120,10 +120,10 @@ test-behat:
 	$(EXEC) vendor/bin/behat $(BEHAT_ARGS)
 
 test-phpunit-functional:                                                                               ## Run phpunit fonctional tests
-	$(EXEC) vendor/bin/phpunit --group functional
+	$(EXEC) vendor/bin/phpunit $(PHPUNIT_ARGS) --group functional
 
 tu: vendor app/config/assets_version.yml                                                               ## Run the PHP unit tests
-	$(EXEC) vendor/bin/phpunit --exclude-group functional
+	$(EXEC) vendor/bin/phpunit $(PHPUNIT_ARGS) --exclude-group functional
```
Comment (tony-tran) [inline Makefile]: Good idea. Maybe you can create a new step just to run phpunit with argument only and use this new step in `tu` ? WDYT ?

**Comment by JeremieSamson** [comment]
@tony-tran makefile updated

File: Makefile [hunk 2]
```diff
@@ -5,6 +5,8 @@ COMPOSER=$(EXEC) composer
 CONSOLE=bin/console
 PHPCSFIXER?=$(EXEC) php -d memory_limit=1024m vendor/bin/php-cs-fixer
 BEHAT_ARGS?=-vvv
+PHPUNIT=$(EXEC) vendor/bin/phpunit
```
Comment (thomasbisignani) [inline Makefile]: Can you add a default args for the `PHPUNIT_ARGS` variable ?
example :  `PHPUNIT_ARGS?=-v`

File: Makefile [hunk 3]
```diff
@@ -5,6 +5,8 @@ COMPOSER=$(EXEC) composer
...
+BEHAT=$(EXEC) vendor/bin/behat
```
Comment (thomasbisignani) [inline Makefile]: Can you add a default args for the PHPUNIT_ARGS variable ?
example : `PHPUNIT_ARGS?=-v`

**Pull request merged**